lib/generators/komponent/templates/styleguide/components/container/komponent_container.css

```diff
@@ -5,13 +5,13 @@
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
     Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
   margin: 40px 60px 0 300px;
+}
 
-  .komponent-code {
-    background-color: #333;
-    color: #fff;
-    margin: 10px 0;
-    padding: 10px;
-  }
+.komponent-footer .komponent-code {
```

Comment:
@nicolas-brousse Why do you add `.komponent-footer`?

Reply:
```suggestion
.komponent-container .komponent-code {
```

Reply:
It's a mistake, too mush copy and paste ...